packages/edit-site/src/components/left-sidebar/navigation-panel/index.js

```diff
@@ -0,0 +1,186 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+import {
+	__experimentalNavigation as Navigation,
+	__experimentalNavigationGroup as NavigationGroup,
+	__experimentalNavigationItem as NavigationItem,
+	__experimentalNavigationMenu as NavigationMenu,
+} from '@wordpress/components';
+import { getBlockType, getBlockFromExample } from '@wordpress/blocks';
+import { BlockPreview } from '@wordpress/block-editor';
+
+const NavigationPanel = () => {
+	const [ activeItem, setActiveItem ] = useState( 'item-1' );
+	const [ activeMenu, setActiveMenu ] = useState( 'root' );
+	const [ showPreview, setShowPreview ] = useState( false );
+
+	return (
+		<>
+			<div style={ { position: 'relative', height: '100%' } }>
```

Comment:
What about adding a `.edit-site-navigation-panel` class here, and move these styles to `styles.scss`?